File: Array 101/Number_disappeared_in_array.cpp
class Solution {
public:
    vector<int> findDisappearedNumbers(vector<int>& nums) {
        int N = nums.size();
        vector<int> result;
        int arr[N+1];
        for(int i=0;i<N+1;i++)
        {
            arr[i]=0;
        }
        for(int i=0;i<nums.size();i++)
        {
            arr[nums[i]]++;
        }
        
        for(int i=1;i<N+1;i++)
        {
            if(arr[i]==0)
             result.push_back(i);
        }
       return result;
    }
};

File: Array 101/Number_disappeared_in_array.py
class Solution:
    def findDisappearedNumbers(self, nums: List[int]) -> List[int]:
        N = len(nums)
        result = []
        arr = []
        for i in range(N+1):
            arr.append(0)
        for j in range(N):
            arr[nums[j]]+=1
        for k in range(1,N+1):
            if arr[k] == 0:
                result.append(k)
        return result
        
        